blocks/countdown/src/countdown-title.tsx

```diff
@@ -16,13 +16,33 @@ export const CountdownTitle: FunctionComponent<CountdownTitleProps> = ({
   const textareaEl = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    if (!textareaEl.current) return;
-    textareaEl.current.textContent = value ?? "";
-  }, [value]);
+    const textarea = textareaEl.current;
+    if (!textarea) return;
+    textarea.textContent = value ?? "";
+
+    // Calling this whenever value changes makes it so the cursor position is lost in some browsers
+    // (firefox and safari for example), so we leave it out of the dependency array
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const textarea = textareaEl.current;
+    if (!textarea) return;
+
+    const onKeyDown = ({ code }: KeyboardEvent) => {
+      if (code === "Enter") {
+        textarea.blur();
+        onBlur();
+      }
+    };
```

Comment:
This can be an `onKeyDown` handler on the element itself. You should also use `key` rather than `code`, because `key === "Enter"` will cover the numpad's enter as well (whereas its code is `NumpadEnter`)
```ts
        onKeyDown={({ currentTarget, key }) => {
          if (key === "Enter") {
            currentTarget.blur();
            onBlur();
          }
        }}
```

Reply:
Changed to use `key` instead of `code` in bfcf717bea86ee97f89575593564186883cbc974

Reply:
Right – this linting rule is flagging a real problem, which is that we're using a `div` as a text input, and if we need to do that we should add additional handling for things that would otherwise be handled natively.

For example:
- add `role="textbox"` to the `div` ([docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role))
- if the user presses `"Escape"`, cancel input

Reply:
Having the key handler done imperatively in an effect doesn't remove these problems, it just makes it impossible for the linter to tell they exist

Reply:
Thank you for the help, I wasn't aware of `role="textbox"`